src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -808,6 +818,12 @@ private static PlatformAttributes CopyAllAttributes(PlatformAttributes copyTo, P
 
                 AddPlatformAttributes(symbol.GetAttributes(), ref attributes);
 
+                if (symbol is IMethodSymbol method && (method.MethodKind == MethodKind.PropertyGet || method.MethodKind == MethodKind.PropertySet))
```

Comment:
What is the containing symbol for this get/set method? I thought it would have been the property symbol, but seems like it was the containing type?

Reply:
yeah it was not, containing symbol was the containing class

Reply:
Thanks, your change seems good.